### ISSUE TITLE
you can't see the button for Sharing/ Tooting from WritePage.js/.ux

### ISSUE THREAD

**Issue by domiSoap** [jeroensmeets/mastodon-app#36]
the button for Sharing a new Post  isn't visible.

**Comment by domiSoap** [comment]
https://www.fusetools.com/docs/fuse/controls/button

i think we should use a design a better one,

this is a example for developing a native button its from fusetools

```html
<Button ux:Class="MyNativeButtonWithFallback" Margin="2">
    <Panel ux:Template="GraphicsAppearance" HitTestMode="LocalBounds">
        <Text Value="{ReadProperty Text}" Color="#fff" Alignment="Center" TextAlignment="Center" Margin="10" />
        <Rectangle CornerRadius="4" Layer="Background" Color="#25a" />
    </Panel>
</Button>
```

**Comment by jeroensmeets** [comment]
fixed in v1.0.7
